### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by christopherasnider**
There was a bit of a backlog...added Done tasks for May, June, and July 2019.

Note: July done tasks have Jira links.

### CODE REVIEW

**Comment by christopherasnider** [comment]
Sorry for the lengthy review, I got behind on logging things and it was too easy to shrug it off week after week.

File: _docs/release-notes.md [hunk 1]
```diff
@@ -24,6 +24,83 @@ Visit [https://app.tidepool.org](https://app.tidepool.org) to see the results of
 
 Don't have a Tidepool account? Visit [tidepool.org/signup](https://tidepool.org/signup) to create your free Tidepool account.  
 
+### 1.23.0 (Released 2019-07-05)
+We're starting off July with a bugfix bonanza. First, we cleaned up some spacing and formating when you edit your profile.
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
We're starting off July with a bugfix bonanza. First, we cleaned up some spacing and formatting when you edit your profile.  
```

File: _docs/release-notes.md [hunk 2]
```diff
@@ -530,6 +607,60 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
 
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
+### 2.17.0 (Released 2019-06-25)
+Tidepool Uploader now supports the Omnipod DASH PDM. Woohoo! Whether you're uploading the original PDM or the DASH PDM, you'll still select "Insulet Omnipod" as your device we'll take care of the rest.
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
Tidepool Uploader now supports the Omnipod DASH PDM. Woohoo! Whether you're uploading the original PDM or the DASH PDM, you'll still select "Insulet Omnipod" as your device and we'll take care of the rest.  
```

File: _docs/release-notes.md [hunk 3]
```diff
@@ -530,6 +607,60 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
...
+We updated the error message that displays when a clinician tries to create a Tidepool account with an email address already associated with an existing Tidepool account. Little things like that are important.  
+[Feature Requirements](https://trello.com/c/7HVJ9gUH/52-bug-uploader-not-handling-the-user-already-exists-error)
+
+Continuing the user experience improvement train, the design of our Mac Installer has improved to be more explicit about the "drag this app into that folder" bit.  
+[Feature Requirements](https://trello.com/c/LpB6l0Sd/53-improve-usability-of-mac-dmg-installer)
+
+And now we're rolling out more test environments to improve our development process, development builds of Tidepool Uploader need to access them.  
+[Feature Requirements](https://trello.com/c/2wTVDU61/54-uploader-add-qa1-environment-to-uploader-server-list)
+
+We created a script that automatically submits a notification to Anti-virus vendors like McAffe and Kaspersky when new versions of Tidepool Uploader are released.
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
We created a script that automatically submits a notification to antivirus vendors like McAfee and Kaspersky when new versions of Tidepool Uploader are released.  
```

File: _docs/release-notes.md [hunk 4]
```diff
@@ -530,6 +607,60 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
...
+Continuing the user experience improvement train, the design of our Mac Installer has improved to be more explicit about the "drag this app into that folder" bit.  
+[Feature Requirements](https://trello.com/c/LpB6l0Sd/53-improve-usability-of-mac-dmg-installer)
+
+And now we're rolling out more test environments to improve our development process, development builds of Tidepool Uploader need to access them.  
+[Feature Requirements](https://trello.com/c/2wTVDU61/54-uploader-add-qa1-environment-to-uploader-server-list)
+
+We created a script that automatically submits a notification to Anti-virus vendors like McAffe and Kaspersky when new versions of Tidepool Uploader are released.  
+[Feature Requirements](https://trello.com/c/Wv9TE21v/55-update-script-to-whitelist-with-kaspersky-and-mcafee-automatically)
+
+We now store Omnipod IBF files in our "blog" service, which helps tremendously when we are troubleshooting reported bugs from our users. (We already do this for Medtronic uploads.)
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
We now store Omnipod IBF files in our "blob" service, which helps tremendously when we are troubleshooting reported bugs from our users. (We already do this for Medtronic uploads.)  
```

File: _docs/release-notes.md [hunk 5]
```diff
@@ -1157,6 +1288,48 @@ OmniPod uploads were running into problems if certain error codes were identifie
 ## Tidepool Mobile  
 Visit https://tidepool.org/mobile to learn more about Tidepool Mobile.  
 
+### iOS - 2.1.12 (Released 2019-07-02)
+We added a tooltip to prompt new Tidepool Mobile users to make a note after they've synced their data.  
+[Feature Requirements](https://tidepool.atlassian.net/browse/MOBILE-9?jql=project%20%3D%20%22MOBILE%22%20AND%20fixVersion%20%3D%20%22iOS%202.1.12%22)
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
[Feature Requirements](https://tidepool.atlassian.net/browse/MOBILE-9)
```

File: _docs/release-notes.md [hunk 6]
```diff
@@ -1157,6 +1288,48 @@ OmniPod uploads were running into problems if certain error codes were identifie
...
+
+And addressed a design issue that prevented users from opening a note created as a result of that first time tooltip prompt.  
+[Feature Requirements](https://tidepool.atlassian.net/browse/MOBILE-10?jql=project%20%3D%20%22MOBILE%22%20AND%20fixVersion%20%3D%20%22iOS%202.1.12%22)
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
[Feature Requirements](https://tidepool.atlassian.net/browse/MOBILE-10)
```

File: _docs/release-notes.md [hunk 7]
```diff
@@ -1157,6 +1288,48 @@ OmniPod uploads were running into problems if certain error codes were identifie
...
+
+[iOS Release](https://appsto.re/us/aXyl9.i)
+
+### Android - 3.0.5 (Released 2019-06-13)
+Technically this work also supports our eventual iOS update to a React Native code base soe both Android and iOS versions of Tidepool Mobile will be using the same code base. This update continues to iterate upon the tooltips that display for fresh installs. We'll get to updating the current iOS release with these tooltips soon.
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
Technically this work also supports our eventual iOS update to a React Native code base so both Android and iOS versions of Tidepool Mobile will be using the same code base. This update continues to iterate upon the tooltips that display for fresh installs. We'll get to updating the current iOS release with these tooltips soon.  
```

**Pull request merged**